src/components/20-molecules/dropdown/index.scss

```diff
@@ -39,6 +39,12 @@ $dropdown-height: 40px;
 axa-dropdown {
   display: block;
 
+  &:not([data-usecase="datepicker"]) {
```

Comment:
mmm.. i kinda dont like this. There is a NOT interface dependency between datepicker and dropdown. I would rather see a non documented property for this purpose. But am also not strong opinionated on it. Is just, we know all kind of interfaces between component are props down and events up. This is kinda a break to the rule

Reply:
And also, data-attributes are not ideal for CSS selection due to its semantics (is just a personal feeling)

Reply:
I reworked a bit, fixing the small-vw issues en passant. No more data- attributes, no more negation.